packages/core/src/common/keybinding.ts

```diff
@@ -26,6 +26,13 @@ export interface Keybinding {
     readonly accelerator?: Accelerator;
 }
 
+export enum KeybindingScopes {
```

Comment:
It should be `KeybindingScopes`, not plural

Reply:
You mean `KeybindingScope` ?